drivers/spi/spi_mcux_flexcomm.c

```diff
@@ -143,6 +147,17 @@ static int spi_mcux_configure(const struct device *dev,
 
 		SPI_MasterGetDefaultConfig(&master_config);
 
+		/* Get the clock frequency */
+		clock_dev = device_get_binding(config->clock_name);
```

Comment:
same issue as in the other driver, run it once at init and keep the clock_dev in device's private data

Reply:
This is run only once in case of the SPI driver. I will fix it in the I2C driver.

Reply:
No spi_mcux_configure will be ran everytime a different spi_config is given to transceive()

that said, clock_freq is probably always the same (or can it be different at runtime?). So either way: either you store clocke_dev in device's private data or, if relevant, clock_freq that you'll get at init time.

Reply:
Got it, thanks